src/Fixture.hpp

```diff
@@ -132,6 +138,34 @@ namespace iodrivers_base
         {
             return driver.getStatus().queued_bytes;
         }
+        void EXPECT_REPLY(std::vector<uint8_t> const& expectation, std::vector<uint8_t> const& reply)
+        {
+            getStream()->EXPECT_REPLY(expectation,reply);
+        }
+
+        /**
+         * Check if the test has any expectation set and throw if positive.
+         * It should be used to check if the test reached its end without 
+         * any expecation left only
+         */
+        void expectationsAreEmpty()
```

Comment:
This should have a verb to describe what the method does. I would suggest `validateExpectationsAreEmpty`